api/camunda/camunda-openapi.yaml

```diff
@@ -0,0 +1,549 @@
+openapi: "3.0.3"
+info:
+  title: Camunda 8 REST API
+  version: "0.1"
```

Comment:
<img width="701" alt="image" src="https://github.com/camunda/camunda-docs/assets/1051363/ba3c8a02-1871-45f1-911b-831edb02e0c8">

This looks a bit awkward. Will 1.0 be released with the 8.6?

Reply:
[this change](https://github.com/camunda/camunda-docs/pull/3919/files#r1644745188) would clean up the duplicate badge issue.

Reply:
(and once that change is made, the badge that remains would be the `Version: Next` one.)

Reply:
This is still happening. Did this change get merged?